INSTALL.md

```diff
@@ -42,7 +42,16 @@ code. The requirements are:
 
 On Windows, it has been reported that installing via `MSYS2` works
 [MSYS2](https://www.msys2.org/). On Windows older than Windows 8, you may need
-to set an environment variable `OLD_WIN=1` or modify it in `config.mk`.
+to set an environment variable `OLD_WIN=1` or modify it in `config.mk`. The idea is that
+this flag makes programs see paths in the old 8.3 convention. Another way to get Idris2
+built on windows is to create a junction without spaces for the ChezScheme base folder, e.g.
+
+```sh
+mklink /j C:\Chez "C:\Program Files\Chez Scheme 9.6.4"
```

Comment:
probably we are bound to manual  handling. Chez Scheme appears to work through several nested cmd calls. I didon't manage to pipe its output into a file; _scheme --version > chezvers.dat_ leaves the file empty; equally I didn't succeed to capture the output in an environment variable. But even if this worked, the junction to the old version should be removed as well as the old path entry. In my opinion trying to fully automate updates is not worth the effort.  The mklink command is well documented by Microsoft (including the command's delete option), and there is no magic: A junction is a small file which points to a folder; although the dir command produces a listing, a junction cannot be used by programs to run through the folder entries normally.
My first idea was to modify the ProgramFiles environment variable to include double quotes. It turned out that this does not work. Anyway, to keep risk low, I used a Windows virtual machine. (You can get a legal copy  of Windows 11 for about 12 $.) Wouldn't this be a possibility to produce a distributable binary copy of the current version and to include this on the download page? I have no idea how much work this would mean in addition to what I experienced - apart from starting the build, the rest takes a couple of minutes which don't require anyone's presence. Running a test suite could be automated as well.

Reply:
Here is a solution: Copy thhese two lines to the file mkChezLink.cmd:

```sh 
@rd C:\Chez
@for /f "tokens=*" %%i in ('dir "%ProgramFiles%\Chez*" /B') do mklink /j C:\Chez "%ProgramFiles%\%%i"
```

Running it will delete an existing junction and create the new one. If Chez Scheme is not installed it does nothing. If the junction does not exist, it will display an error.

Reply:
In the `for`command, a delimiter needs to be added. Otherwise, folder names are extracted only up to the first space. Hence:
```sh
@for /f "tokens=1 del=:" %%i in ('dir "%ProgramFiles%\Chez*" /B /O-D') do mklink /j C:\Chez "%ProgramFiles%\%%i"
````
I corrected this in my fork.